modules/oe_translation_cdt/oe_translation_cdt.module

```diff
@@ -60,3 +96,102 @@ function oe_translation_cdt_oe_translation_request_view_alter(array &$build, Ent
     ];
   }
 }
+
+/**
+ * Implement template_preprocess_table__remote_language_list().
+ */
+function oe_translation_cdt_preprocess_table__remote_language_list(array &$variables) {
+  $id = (string) $variables['attributes']['data-translation-request'];
+  $request = TranslationRequestCdt::load($id);
+  if (!$request || $request->bundle() !== 'cdt') {
+    return;
+  }
+
+  foreach ($variables['rows'] as &$row) {
+    // Add the accepted deadline per individual language.
+    $langcode = (string) $row['attributes']['hreflang'];
+    $language = $request->getTargetLanguage($langcode);
+
+    if ($language->getStatus() !== TranslationRequestRemoteInterface::STATUS_LANGUAGE_REVIEW) {
+      continue;
+    }
+
+    $row['cells']['operations']['content']['#links'][] = [
+      'title' => 'Fetch the translation',
+      'url' => Url::fromRoute('oe_translation_cdt.fetch_translation', [
+        'translation_request' => $id,
+        'langcode' => $langcode,
+      ], [
+        'query' => [
+          'destination' => Url::fromRoute('<current>')->toString(),
+        ],
+      ]),
+    ];
+  }
+
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function oe_translation_cdt_oe_translation_request_presave(TranslationRequestCdtInterface $request) {
```

Comment:
If both the cdt and poetry module are enabled, this will fail because poetry will save entities with a different interface. The paramer type should be TranslationRequestInterface and then we check whether it implements the cdt interface